tests/test-empress.js

```diff
@@ -353,6 +353,122 @@ require([
             }
         });
 
+        test("Test colorByFeatureMetadata, continuous", function () {
+            // get color map
+            var cm = this.empress.colorByFeatureMetadata(
+                "f1",
+                "Viridis",
+                "tip",
+                false,
+                true
+            );
+            var properties = [
+                "gradientID",
+                "gradientSVG",
+                "maxValStr",
+                "midValStr",
+                "minValStr",
+                "missingNonNumerics",
+                "pageSVG",
+            ];
+
+            // make sure color gradient returns correct keyInfo
+            var resultProperties = util.naturalSort(Object.keys(cm));
+            deepEqual(resultProperties, properties);
+
```

Comment:
```suggestion

            deepEqual(cm.gradientID, "Gradient-1");
            deepEqual(cm.maxValStr, "2");
            deepEqual(cm.midValStr, "1.5");
            deepEqual(cm.minValStr, "1");
            deepEqual(cm.missingNonNumerics, false);
```

(While we're at it, may as well test a few easy things.)